README.md

```diff
@@ -118,6 +118,36 @@ Default: `1`
 
 How many releases of changelog you want to generate. It counts from the latest semver tag. Useful when you forgot to generate any previous releases. Set to `0` to regenerate all.
 
+##### host
+
+Default: `api.github.com`
+
+Github API hostname
+
+##### pathPrefix
+
+Default: `none`
+
+Github API pathPrefix 
+
+##### protocol
+
+Default: `https`
+
+Github API protocol 
+
+##### port
+
+Default: `80 (http), 443 (https)`
+
+Github API port
+
+#### context
```

Comment:
But these are above the context inside the options key. How one is going to know they can configure to which github server they want to push to?